src/edge-cs-coreclr/global.json

```diff
@@ -0,0 +1,7 @@
+{
+    "sdk": {
+        "version": "1.0.0-beta7-12331",
```

Comment:
I'm honestly not sure what this section of the global.json is even used for, it certainly doesn't have any effect on the compilation.  I think it's used by Visual Studio when it does package restores and Intellisense and needs to know which version of the framework to use.  That being said, I've gone ahead and updated the version for consistency's sake.